src/backend/common/graphics_common.cpp

```diff
@@ -382,12 +382,6 @@ forge::Image* ForgeManager::getImage(forge::Chart* chart, int w, int h,
 forge::Plot* ForgeManager::getPlot(forge::Chart* chart, int nPoints, forge::dtype dtype,
                                    forge::PlotType ptype, forge::MarkerType mtype)
 {
-    /* nPoints needs to fall in the range of [0, 2^48]
-     * for the ForgeManager to correctly retrieve
-     * the necessary Forge Plot object. So, this implementation
-     * is a limitation on how big of an plot graph can be rendered
-     * using arrayfire graphics funtionality */
-    assert(nPoints <= 2ll<<48);
```

Comment:
We should put this limitation somewhere easily accessible for users, may be in documentation. I can understand why you would remove assert statements, but why remove the comments ?

Reply:
It was removed because nPoints is an int and that assertion will always return true.